save.go

```diff
@@ -15,6 +15,7 @@ import (
 func saveImage(image imgutil.Image, additionalNames []string, logger Logger) (platform.ImageReport, error) {
 	var saveErr error
 	imageReport := platform.ImageReport{}
+	logger.Infof("Publishing %s\n", image.Name())
```

Comment:
I thought about listing all the image names like we do in the `*** Images` section. I also thought about a more generic message like `Pushing image(s)...`. Let me know what you think!

Reply:
Maybe it's better that we'll print the same value as we print in the following line: https://github.com/buildpacks/lifecycle/blob/203221553c114ecbad7378e16ff02385dd86d23d/save.go#L34
I think that printing `image.Name()` is clearer but maybe there is a reason that we print `shortID(id)` in the other place.

Reply:
Will the `id` be available via `image.Identifier()` _before_ the push? I thought that was likely invalid to use until a push to a registry had occurred.

Reply:
ohhhh interesting. I'm not sure :\
If it's not, maybe you can just add `image.Name` to the print after the push (in addition to `shortID(id)`).
I think that in this way it'll be easier to the reader to see the progress.

Reply:
The output already has the name it looks like
```
Publishing 192.168.86.20:5005/myrepo/img **new line
*** Images (sha256:cb66dc1599dd7be223b5dc3e88dc9f2eef1aec0aaf609c56aef2cb3ee394ff84):
      192.168.86.20:5005/myrepo/img
```

Reply:
All the tags are output for the image that was pushed, so I think it lines up ok.

Reply:
You're right. I missed that we're printing more information after https://github.com/buildpacks/lifecycle/blob/203221553c114ecbad7378e16ff02385dd86d23d/save.go#L34
Thanks for the clarification!